### PR TITLE
feat(screenshare): Audio only screenshare

### PR DESCRIPTION
**Pull request by andrei-gavrilescu**

### CODE REVIEW

File: react/features/toolbox/components/web/Toolbox.js [hunk 1]
```diff
@@ -983,6 +999,13 @@ class Toolbox extends Component<Props> {
                 && <SharedVideoButton
                     key = 'sharedvideo'
                     showLabel = { true } />,
+            this._shouldShowButton('shareaudio') && isScreenAudioSupported()
+                && <OverflowMenuItem
+                    accessibilityLabel = { t('toolbar.accessibilityLabel.shareaudio') }
+                    icon = { IconShareAudio }
+                    key = 'shareaudio'
```
Comment (jallamsetty1) [inline react/features/toolbox/components/web/Toolbox.js:1006]: I think the button name "Share audio" is little confusing. Do you think `Start Audio Share` and `Stop Audio Share` make more sense here ?

**Review by jallamsetty1 — COMMENTED** [review]
@andrei-gavrilescu This LGTM overall, left 2 minor comments.

File: conference.js [hunk 2]
```diff
@@ -1660,6 +1664,23 @@ export default {
                 = this._turnScreenSharingOff.bind(this, didHaveVideo);
 
             const desktopVideoStream = desktopStreams.find(stream => stream.getType() === MEDIA_TYPE.VIDEO);
+            const dekstopAudioStream = desktopStreams.find(stream => stream.getType() === MEDIA_TYPE.AUDIO);
+
+            if (dekstopAudioStream) {
```
[the remark this answers is not in the document]
Comment (andrei-gavrilescu) [reply conference.js:1669]: Imo this behaviour is fine. If the user initiates a screen sharing session and goes through the trouble of ticking the Share audio box and there's a problem with that audio track at some point, then the behaviour wouldn't be as expected same as what would happen if something went wrong with the video track.
Of course this is just my point of view, if you feel strongly about it we can think of another way to approach this scenario.

File: react/features/toolbox/components/web/Toolbox.js (hunk 1 above)
Comment (andrei-gavrilescu) [reply react/features/toolbox/components/web/Toolbox.js:1006]: As it stands the button doesn't have a state it's just Share audio for both stop and start. We're planning on adding a "How to" dialog when the user clicks that button and add that state then. This is meant to be an initial iteration. But if you think it's something we must have for this PR, I can add that state now.

**Review by jallamsetty1 — APPROVED** [review]

**Review by vp8x8 — APPROVED** [review]

**Pull request merged**